README.md

```diff
@@ -10,51 +10,52 @@ There are two main categories of samples: **camera** and **applications**. The s
   - **advanced**
     - [**hand_eye_calibration**][hand_eye_calibration-url]
-      - [**calibrate_eye_to_hand**][calibrate_eye_to_hand-url] - This samples shows how to perform a complete Hand to Eye calibration
-      - [**utilize_eye_in_hand_calibration**][utilize_eye_in_hand_calibration-url] - Transform a 3D point from camera frame to robot base frame using hand-eye calibration matrix.
-      - [**pose_conversions**][pose_conversions-url] - Convert to/from Transformation Matrix (Rotation Matrix + Translation Vector)
+      - [**calibrate_eye_to_hand**][calibrate_eye_to_hand-url] - Perform Eye-to-Hand calibration.
+      - [**utilize_eye_in_hand_calibration**][utilize_eye_in_hand_calibration-url] - Transform a single data point or entire point cloud from the camera frame to the robot base frame using the Eye-in-Hand calibration matrix.
+      - [**pose_conversions**][pose_conversions-url] - Convert to/from Transformation Matrix (Rotation Matrix + Translation Vector).
       - **universal_robots_hand_eye_calibration**
-        - [**universal_robots_perform_hand_eye_calibration**][ur_perform_hand_eye_calibration-url] - Generate dataset and perform hand-eye calibration on generated dataset.
+        - [**universal_robots_perform_hand_eye_calibration**][ur_perform_hand_eye_calibration-url] - Generate dataset and perform Hand-Eye calibration on the generated dataset.
         - **Dependencies:**
           - [OpenCV](https://opencv.org/) version 4.0.1 or newer.
           - [RTDE][rtde_guide-url] version 1.0 or newer.
           - [Scipy](https://www.scipy.org/) version 1.4.0 or newer.
-    - [**downsample**][downsample-url]  - This example shows how to import a Zivid point cloud from a .ZDF file and downsample it.
+    - [**downsample**][downsample-url]  - Downsample point cloud from a ZDF file.
       - **Dependencies:**
         - [Eigen](http://eigen.tuxfamily.org/) version 3.3.7 or newer
-    - [**create_depth_map**][create_depth_map-url] - Import a ZDF point cloud and convert it to OpenCV format, then extract and visualize depth map.
-    - [**mask_point_cloud**][mask_point_cloud-url] - Import ZDF point cloud, apply a binary mask, and visualize it.
+    - [**create_depth_map**][create_depth_map-url] - Read point cloud data from a ZDF file, convert it to OpenCV format, then extract and visualize depth map.
+    - [**mask_point_cloud**][mask_point_cloud-url] - Read point cloud data from a ZDF file, apply a binary mask, and visualize it.
       - **Dependencies:**
         - [OpenCV](https://opencv.org/) version 4.0.1 or newer
-    - [**gamma_correction**][gamma_correction-url] - Modify gamma of the 2D image.
+    - [**gamma_correction**][gamma_correction-url] - Modify gamma of 2D image.
       - **Dependencies:**
         - [OpenCV](https://opencv.org/) version 4.0.1 or newer
-    - [**color_balance**][color_balance-url] - Balance color for RGB image.
+    - [**color_balance**][color_balance-url] - Balance color of 2D image.
```

Comment:
It is a bit unclear how to use this sample. Am I supposed to point at a white surface?

Reply:
Yes, nothing except a white surface should be in the scene.

Reply:
Is it worth mentioning that to the user somewhere?

Reply:
I've added another commit that specifies the tested version of Python and also required versions for other libraries.

Reply:
I think you wrote something wrong here :)
`Note: The version tested with Zivid cameras 3.7.7.`

Reply:
Ah you fixed it.